shippable.yml

```diff
@@ -583,12 +583,27 @@ jobs:
       - IN: mktg_img
       - IN: micro_img
       - IN: u16admiral_img
+      - IN: shipit_bits_cli
+        scopes:
+          - ecr
+        switch: off
       - TASK:
           name: deploy_to_rc
           script:
             - pushd $(shipctl get_resource_state "shipit_repo")
             - ./deployRc.sh
             - popd
+      - TASK:
+          name: cleanup_untagged_images
+          script:
+            - |
+              repositories=$(aws ecr describe-repositories --output text | awk '{print $5}')
```

Comment:
can we add some print statements here to list the number of repos found.

Reply:
ok i will print the untagged image that is getting deleted and the repository it is going to get deleted from